slackClient.js

```diff
@@ -65,29 +42,29 @@ class SlackClient {
 
   /**
    * @param {string} channel
-   * @param {string} title
-   * @private
+   * @param {string} text
+   * @param {Object} options
    */
-  _searchAnime(channel, title) {
-    MyAnimeList.getAnime(title)
-      .then(response => response ?
-        this.slack.chat.postMessage(this._buildAnimeSlackResponse(channel, response)) :
-        this._postMessage(channel, `Did not find anime with title "${title}"`)
-      )
+  postMessage(channel, text, options) {
+    this.slack.chat.postMessage({
+      channel,
+      text,
+      ...options
+    })
       .catch(err => {
-        throw new Error(`Search Anime Error: ${err}`);
+        throw new Error(`Slack Post Message Error: ${err}`);
       });
   }
 
   /**
-   * @param {string} channel
+   * Formats anime object for slack chat postMessage format
+   *
    * @param {Anime} anime
-   * @returns {{channel: *, attachments: *[]}}
-   * @private
+   * @returns {{attachments: *[]}}
    */
-  _buildAnimeSlackResponse(channel, anime) {
+  buildAnimeSlackMessage(anime) {
```

Comment:
I believe this should be done in the index.js file and abstracted outside of the slack client. We don't want to have any anime-related logic or formatting in the SlackClient at all. It should just be a dummy class that connects to slack, posts messages, and binds to slack events

Right now we have 2 modules and 1 main program. The 2 modules are the AnimeDataClient and the SlackClient. The main program is the procedure that will be orchestrating the interaction between these two different abstractions, or else we're mixing concerns and tainting the purity of the abstractions. What do you think?